Patch provided by Michael Bienia <michael@bienia.de> for 0.7.2-1, see #539540

--- freqtweak-0.7.2/src/FTutils.hpp	2009-07-11 23:15:00.000000000 +0200
+++ src/FTutils.hpp	2009-12-07 18:05:09.000000000 +0100
@@ -23,6 +23,7 @@
 #include <iostream>
 #include <sstream>
 #include <cmath>
+#include <stdint.h>
 
 //#include <wx/string.h>
 
